web/concrete/src/Workflow/Request/PageRequest.php

```diff
@@ -35,7 +35,7 @@ public function setRequestedPageVersionID($cvID)
         $this->cvID = $cvID;
     }
 
-    public function addWorkflowProgress(\Concrete\Core\Workflow\Workflow $wf)
```

Comment:
Hah, well, now I see why you did it. Regardless, it ought to be the original workflow class rather than the alias. I'll make the change everywhere.